src/nvim/ops.c

```diff
@@ -3719,7 +3719,7 @@ char_u *skip_comment(
 // to set those marks.
 //
 // return FAIL for failure, OK otherwise
-int do_join(size_t count,
+int do_join(ptrdiff_t count,
```

Comment:
This was enough to get a screen diff (no trailing garbage past NULL) for OpenBSD in https://github.com/neovim/neovim/pull/12611. Is there a heisenbug?

Reply:
No, it's likely a crash.